lib/rules/submit-button.json

```diff
@@ -0,0 +1,17 @@
+{
+  "id": "submit-button",
+  "selector": "form[action][method]",
+  "tags": [
+    "wcag2a",
+    "wcag322",
+    "cat.forms",
+    "experimental"
+  ],
+  "metadata": {
+    "description": "Ensure that forms can be submitted by using submit buttons",
+    "help": "Forms with action and method must have an way to be submited"
```

Comment:
I think this is describing the wrong problem. The problem isn't that forms need a submit mechanism, the problem is that users lose their focus if forms automatically submit after user input.